### DISCUSSION TITLE
worker without complete job

### DISCUSSION THREAD

**Discussion by cx-eyal-goren** [camunda-community-hub/pyzeebe#275] [Q&A]
i want to make worker that get tasks by task type
and i dont want to complete the task when i get it

i am sending the task to other services and when that r done 
i want to complete the task from another function

readTasks() --> sendToProccess(job.key)--->finishTask(job.key)

is this possible with this client?

**Comment by kbakk** [comment]
You could look into overriding `Job.set_success_status​` to not do the complete job call in a custom task decorator. 

See the relevant code you'd override here: 

https://github.com/camunda-community-hub/pyzeebe/blob/dd543ea2d05bc73483b43da97e2d8dbfc6043847/pyzeebe/job/job.py#L55 

Not sure this would be supported in the future though, see the discussion in https://github.com/camunda-community-hub/pyzeebe/issues/179.
